modules/oc-mirror-procedure-delete-v2.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-procedure-delete-v2_{context}"]
+= Procedure for deleting the images from disconnected environment
```

Comment:
No lead-in sentence or ".Procedure" subheading. This would be the mod doc standard.